### ISSUE TITLE
Can we add space between glossary items?

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/styles#23]
![gloss](https://cloud.githubusercontent.com/assets/1506457/6790000/384c980c-d183-11e4-8097-d4d6517ac71f.jpg)

**Comment by kynan** [comment]
Any news on this?

**Comment by r-gaia-cs** [comment]
@kynan We will make change on the style only after 5.3. If you want to send a pull request it is welcome.

**Comment by kynan** [comment]
In fact I think this is already done, this is how it looks for me now:
![Glossary](https://cloud.githubusercontent.com/assets/346079/7556025/2c247798-f758-11e4-9ee9-216a7bfbee50.png)

**Comment by r-gaia-cs** [comment]
@kynan From which lesson is the screenshot?

**Comment by kynan** [comment]
python-novice-inflammation

**Comment by r-gaia-cs** [comment]
~~~
$ git rev-parse HEAD
e3bda197155dbb093ada96298386396c470e584d
$ make -B preview
$ firefox reference.html
~~~

gives me

![screen shot 2015-05-10 at 19 30 01](https://cloud.githubusercontent.com/assets/1506457/7556535/2afcd14c-f74b-11e4-943c-18733f3702ec.png)

**Comment by kynan** [comment]
Repeating the above with python-novice-inflammation@c97d543 I get the spacing in both Firefox 37.0.2 and Chromium 41.0.2272.76 with html generated by pandoc 1.12.2.1.
